test/unit/visitor/sympy_solver.cpp

```diff
@@ -65,6 +65,22 @@ std::vector<std::string> run_sympy_solver_visitor(
     return results;
 }
 
+// check if in a list of vars (like LOCAL) there are duplicates
+bool is_unique_vars(std::string result) {
```

Comment:
I will need bit some more info on this tomorrow - I wonder why there will be name clash.

Reply:
if we do not replace correctly in `sympy_solver`, using the correct routine, we can get name clashes. For example if I have:
```
DERIVATIVE {
LOCAL old_x
x' = ...
y' = ...
}
```

if I forgot something it could be that 2 `old_x` appear in local